src/styles.ts

```diff
@@ -0,0 +1,142 @@
+import styled, { createGlobalStyle } from "styled-components";
+import { HashLink } from "react-router-hash-link";
+import img from "./highlighter.png";
+
+export const GlobalStyle = createGlobalStyle`
+    body {
+        font-family: 'Josefin Sans', sans-serif;
+        font-weight: 300;
+        font-size: 18px;
+        };
+
+    h1 {
+        font-weight: 700;
+        font-size: 45px;
+        text-shadow: 4px 4px 5px #9ab0af;
+        align-self: center;
+        text-align: center;
+        margin-bottom: 5px;
+
+        @media screen and (max-width: 1668px) {
+          font-size: 40px;
+         }
+    };
+
+    h2 {
+        font-weight: 700;
+        font-size: 25px;
+
+        @media screen and (max-width: 1668px) {
+          font-size: 18px;
+         }
+    };
+
+    h3 {
+        font-weight: 400;
+        font-size: 18px;
+    };
+    
+    h4 {
+        font-weight: 400;
+        font-size: 20px;
+    }
+`;
+
+export const theme = {
+  colors: {
+    white: "#fff",
+    primaryUi: "#EBEBEB",
+    secondaryUi: "rgba(200, 214, 213, .9)",
+    tertiaryUi: "#9AB0AF",
+    fourthUi: "#23362F",
+    fifthUi: "#D9D9D9",
+    sixthUi: "#FAFF05",
```

Comment:
it's might be better to have more meaningful names for those colors, like `divider`,`uiBackground` etc